ros2component/ros2component/verb/standalone.py

```diff
@@ -16,8 +16,8 @@
 import signal
 import uuid
 
+from ros2cli.node.direct import add_arguments
 from ros2cli.node.direct import DirectNode
-from ros2cli.node.strategy import add_arguments
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.

Reply:
Unrelated, yes. Reverted in https://github.com/ros2/ros2cli/pull/494/commits/69363670e4c14ede0a1cd48e80cd4b4dce4fed2f. Same for all other cases below.